pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -0,0 +1,114 @@
+package bundlesecret
+
+//export OPENSHIFTCONFIG=./openshift.local.certificates/admin/.kubeconfig
+import (
+	"fmt"
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const MaxSecretSize = 1 * 1024 * 1024
+
+func NewCmdBundleSecret(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
+	var sourcelist util.StringList
+	cmd := &cobra.Command{
+		Use:   "bundle-secret NAME -f SOURCENAME",
+		Short: "bundle data from a source list to load kubernetes secret",
+		Long:  "bundle data from a source list to load kubernetes secret",
+		Run: func(cmd *cobra.Command, args []string) {
+			var err error
+			if len(args) == 0 {
+				err = fmt.Errorf("Must specify NAME as an argument")
+				checkErr(err, out)
+			}
+			objname := args[0]
```

Comment:
Go style convention is for camelCase variable names.

objname should be objName
secretdata should be secretData

Recommend reading:
https://golang.org/doc/effective_go.html

See section on MixedCaps